src/interfaces/IMetaMorpho.sol

```diff
@@ -145,6 +147,7 @@ interface IMetaMorphoBase {
     /// @notice Warning: Anyone can supply on behalf of the vault so the call to `updateWithdrawQueue` that expects a
     /// market to be empty can be griefed by a front-run. To circumvent this, the allocator can simply bundle a
     /// reallocation that withdraws max from this market with a call to `updateWithdrawQueue`.
+    /// @dev Warning: Removing a market with supply will decrease the fee accrued until the next deposit/withdrawal.
```

Comment:
If the amount removed is bigger than the total amount of interest accrued by the other markets, the fee to accrue will be zero. It's worth mentioning it in the comment or anyway be more specific.

Reply:
can do it yes even though I think this is the general case